README.md

```diff
@@ -0,0 +1,74 @@
+# ostext-style-guide
+
+Design Guide is an application used by OpenStax as a living style guide document for our book templates.
+
+## Requirements
+You will need the following properly installed on your computer.
+
+* [Git](http://git-scm.com/)
+* [Node.js](http://nodejs.org/) (with NPM)
+* [Python 2.7](https://www.python.org/downloads/) or above
+  * If using version 2.7 then you must install [subprocess32](https://pypi.python.org/pypi/subprocess32)
+  * Download zip file then run `python setup.py install` inside of unzipped downloaded folder
+* Gulp CLI `npm install gulp-cli -g`
```

Comment:
Does this need to be installed globally? I think that this is just a _recommendation_ from gulp. 

For example, https://github.com/Connexions/webview and https://github.com/openstax/os-webview both use different versions of `gulp-cli` since they were developed several years apart.